### PR TITLE
Update 2.x changelog and create 2.7.0 release notes

### PR DESCRIPTION
**Pull request by joshuarrrr**
### Description

Update and correct previous changelog sections
Add new entries for 2.7.0 release
Duplicate unreleased section of changelog to 2.7.0 release notes file

Reviewers - please review and comment on the CHANGELOG - any changes will be made there first, and then copied to the release notes.

### Issues Resolved

N/A

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,59 +7,117 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+- [Multiple DataSource] Refactor dev tool console to use opensearch-js client to send requests ([#3544](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3544))
+- [Multiple DataSource] Present the authentication type choices in a drop-down ([#3693](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3693))
+- [Table Visualization] Move format table, consolidate types and add unit tests ([#3397](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3397))
 
 ### 🔩 Tests
 
+- Update caniuse to 1.0.30001460 to fix failed integration tests ([#3538](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3538))
+- [Tests] Fix unit tests for `get_keystore` ([#3854](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3854))
+
+## [2.6.0]
```
Comment (ashwin-pc) [inline CHANGELOG.md:92]: Are you sure this is right? 2.6 looks surprisingly bare. I'm assuming that this is a draft and thats why

File: CHANGELOG.md [hunk 2]
```diff
@@ -186,7 +244,6 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multi DataSource] UX enhancement on Index Pattern management stack ([#2505](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2505))([#2527](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2527))
 - [Multi DataSource] UX enhancement on Update stored password modal for Data source management stack ([#2532](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2532))
 - [Plugin Helpers] Facilitate version changes ([#2398](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2398))
-- [Vega] Add Filter custom label for opensearchDashboardsAddFilter ([#3640](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3640))
```
Comment (ashwin-pc) [inline CHANGELOG.md:189]: Why did we remove these from 2.5?

Comment (joshuarrrr) [reply CHANGELOG.md:189]: It had been added to the wrong section - it wasn't in 2.5.0, it's being released for the first time in 2.7.0: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3640

File: release-notes/opensearch-dashboards.release-notes-2.7.0.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- [TSVB] Fix typo in TSVB README ([#3518](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3518))
+- [UI Actions] Improve UI actions explorer ([#3614](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3614))
+
+### 🛠 Maintenance
+
+- Relax the Node.js requirement to `^14.20.1` ([#3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
+- Bump the version of Node.js installed by `nvm` to `14.21.3` ([#3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
+- Allow selecting the Node.js binary using `NODE_HOME` and `OSD_NODE_HOME` ([#3508](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3508))
+- Remove the unused `renovate.json5` file ([#3489](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3489))
+- Bump `styled-components` from 5.3.5 to 5.3.9 ([#3678](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3678))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-2.7.0.md:72]: Do we want to update this to: Bump `styled-components` from `5.3.5` to `5.3.9` to make it consistent to other bumps?

Comment (joshuarrrr) [reply release-notes/opensearch-dashboards.release-notes-2.7.0.md:72]: yeah - I missed that one

Comment (joshuarrrr) [reply release-notes/opensearch-dashboards.release-notes-2.7.0.md]: fixed

**Review by abbyhu2000 — APPROVED** [review]

**Pull request merged**